foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/platform/database/Oracle10Platform.java

```diff
@@ -38,4 +38,16 @@ protected String buildFirstRowsHint(int max){
         //bug 374136: override setting the FIRST_ROWS hint as this is not needed on Oracle10g
         return "";
     }
+    
```

Comment:
Update the dates in the header to be 2019. This review comment applies to all the files your changing